virttest/qemu_vm.py

```diff
@@ -1205,6 +1205,9 @@ def add_boot(devices, boot_order, boot_once, boot_menu, boot_strict):
                 cmd += " %s" % ",".join(options)
             else:
                 cmd = ""
+            if params.get("seabios_reboottimeout"):
+                reboot_timeout = " reboot-timeout=%s" % params.get('seabios_reboottimeout', 60)
+                cmd += reboot_timeout
```

Comment:
https://github.com/avocado-framework/avocado-vt/blob/cfe37e740913fd9cc8ad6657b3e3474fc1f747aa/virttest/qemu_vm.py#L2112-L2119
1. This option is used by `-boot`, so the param prefix should be `boot_` not `seabios_`.

2. L1209 has an issue when it work with other boot options, please fix that.

Reply:
@luckyh  Thanks your comments.   I have updated this part of code.  please help review it again.